tests/functional/adapter/custom_schema_tests/seeds.py

```diff
@@ -0,0 +1,50 @@
+#
+# Seed queries
```

Comment:
Shouldn't these be done with actual seed tables? dbt["seed", "--full-refresh"] will refresh the table, and can be run before each test case.

Reply:
Not sure I follow. The setUp method should create the tables in the project scope.

Reply:
You can create a csv from text, like here:
https://github.com/dbt-labs/dbt-redshift/blob/4cc967c966a89a1652ed00d051c7b06d348a0925/tests/functional/adapter/test_snapshot.py#L17

register it as a seed, like here:
https://github.com/dbt-labs/dbt-redshift/blob/4cc967c966a89a1652ed00d051c7b06d348a0925/tests/functional/adapter/test_snapshot.py#L90

and then load it in on setup, like here:
https://github.com/dbt-labs/dbt-redshift/blob/4cc967c966a89a1652ed00d051c7b06d348a0925/tests/functional/adapter/test_snapshot.py#L99

I wrote those tests so that I would never need to reload the seed, so it's just `run_dbt(["seed"])` as a class fixture. But if you wanted to reload after each test method, you could do `run_dbt(["seed", "--full-refresh"])` as a method fixture ("function" in this context) and it will repopulate the seed tables.

Reply:
That's a good point. I figured there might be some other reason not to use the customary seeds fixture (perhaps order or something like that?). However, Mike, you're right to call out what you have if there's no reason to manually create them via queries.